qiita_db/test/test_util.py

```diff
@@ -470,11 +470,19 @@ def test_get_mountpoint(self):
         self.assertEqual(obs, exp)
 
     def test_get_files_from_uploads_folders(self):
-        # something has been uploaded
+        # something has been uploaded and ignoring hidden files/folders
+        # and folders
         exp = ['uploaded_file.txt']
         obs = get_files_from_uploads_folders("1")
         self.assertEqual(obs, exp)
 
+        # something has been uploaded and not ignoring hidden files/folders
+        # and folders
+        exp = ['.hidden_file.txt', 'a_folder', 'uploaded_file.txt']
+        obs = get_files_from_uploads_folders("1",
+                                             ignore_hidden_and_folders=False)
+        self.assertEqual(sorted(obs), sorted(exp))
```

Comment:
You should use [`assertItemsEqual`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertItemsEqual).